Makefile

```diff
@@ -286,3 +297,24 @@ ebpf-generate: prereqs ## Generating BPF Go bindings
 docker-generate: ## Creating the container that generates the eBPF binaries
 	docker build . -f hack/generators.Dockerfile -t $(LOCAL_GENERATOR_IMAGE)
 	docker run --rm -v $(shell pwd):/src $(LOCAL_GENERATOR_IMAGE)
+
+##@ Daemon development
+.PHONY: daemon
+daemon: ## Build the daemon.
+	hack/build-daemon.sh
+
+.PHONY: docker-build-daemon
```

Comment:
can u invoke those rules from docker-build and docker-push commands so we do one command to build all ?

Reply:
done